stdlib/ast.pyi

```diff
@@ -34,6 +34,12 @@ class _Attributes(TypedDict, Generic[_EndPositionT], total=False):
 # but they consider themselves to live in the ast module,
 # so we'll define the stubs in this file.
 class AST:
+    @overload
+    def __init__(self) -> None: ...
```

Comment:
This probably doesn't do much in practice since nobody is constructing the base AST class. We could add this sort of overloads to every AST subclass below but I'm not sure that would make users' lives better.

Reply:
> but I'm not sure that would make users' lives better

Same, personally I wouldn't mind just closing this PR and https://github.com/python/typeshed/issues/4760 . I mostly opened it to leave a trace.